RankensteinApp/App.js

```diff
@@ -296,7 +753,31 @@ class CreateAccountScreen extends React.Component {
         <TouchableOpacity
                style = {styles.submitButton}
                onPress = {
-                  () => this.createAccount(this.state.username, this.state.password,this.state.verifypass,this.state.email)
+                 () => {
+					var userSuccess = false
```

Comment:
See if you can enable or disable this button automatically given the current state of the inputs. For example, if the password is malformed or hasn't been entered, don't let the button be pressable. Also, if the inputs are valid and the user makes a change that makes any input malformed, disable the button again.

You can make this a future change.